restlight-test/restlight-test-starter/src/main/java/io/esastack/restlight/test/bootstrap/Restlight4SpringTest.java

```diff
@@ -47,7 +51,8 @@ protected Deployments4SpringTest createDeployments() {
     }
 
     @Override
-    protected RestlightServer doBuildServer(RestlightHandler<RequestContext> handler) {
-        return new FakeServer(handler);
+    protected RestlightServer doBuildServer(RestlightHandler<RequestContext> handler,
+                                            List<InternalFilter<FilterContext>> fs) {
+        return new FakeServer(handler, fs, deployments().filterContext(), deployments().exceptionHandler());
```

Comment:
Why not use `BaseRestlightServer.buildServer` directly?

Reply:
Because FakeServer needs to build in this method.